src/components/android-settings-container.js

```diff
@@ -39,12 +44,18 @@ class AndroidSettingsContainer extends React.Component {
      */
     handleChange = e => {
         e.preventDefault()
-        if (e.target.name === 'valuesTEI') {
-            const valueInput = e.target.value
-            e.target.value > maxValues.valuesTEI
-                ? (e.target.value = maxValues.valuesTEI)
-                : (e.target.value = valueInput)
+
+        let { value } = e.target
+        e.target.name === 'encryptDB'
+            ? (value = e.target.checked)
+            : (value = e.target.value)
+
+        if (e.target.name === 'reservedValues') {
+            value > maxValues.reservedValues
+                ? (value = parseInt(maxValues.reservedValues))
```

Comment:
`maxValues.reservedValues` is a number (`50`), so you don't need to call `parseInt` on it